modules/solid_mechanics/src/materials/SolidModel.C

```diff
@@ -989,7 +995,10 @@ SolidModel::crackingStrainDirections()
   {
     // Adjust the elasticity matrix for cracking.  This must be used by the
     // constitutive law.
-    _local_elasticity_tensor->adjustForCracking( _crack_flags_local );
+    if (_compute_method >= 2)
```

Comment:
Uhm, no. If you implement a new method you will surely have to adjust this if as well, right? Might as well make it a switch statement with a `default:` clause that throws a mooseError